src/TR_alg.jl

```diff
@@ -250,7 +249,7 @@ function TR(
       (has_bounds(f) || subsolver == TRDH) ?
       set_bounds!(ψ, max.(-Δk, l_bound - xk), min.(Δk, u_bound - xk)) : set_radius!(ψ, Δk)
     end
-    ν = 1 / (νInv + 1 / (Δk * α))
+    ν = 1 / (λmax + 1 / (Δk * α))
```

Comment:
```suggestion
    αΔ = α * Δk
    ν = αΔ / (1 + λmax * (1 +  αΔ))
```
?

Reply:
I don't understand. That's the change I committed already.

Reply:
You did not change it inside the while loop.

Reply:
Oh duh.